app/js/controllers/events.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+angular
+.module('app.controllers')
+.controller('EventsCtrl', function($scope, $location, Restangular) {
+	$scope.loadingPromise = Restangular.one('events/next-10-publicly')
```

Comment:
Cool, good to know. So you're saying that we can replace this route with `/upcoming-publicly?page=0`? I am a bit confused about the URL format that you specified in that commit. How do we specify the limit there?